docs/infrastructure/deployment-targets/linux/index.md

```diff
@@ -8,6 +8,7 @@ hideInThisSectionHeader: true
 Linux servers can be configured as [deployment targets](/docs/infrastructure/deployment-targets/index.md) in Octopus.   
 
 The Octopus Server can communicate with Linux targets via SSH or the [Linux Tentacle](https://octopus.com/docs/infrastructure/deployment-targets/linux/tentacle).  
+The Octopus Server can communicate with Linux targets via SSH or the [Linux Tentacle](https://octopus.com/docs/infrastructure/deployment-targets/linux/tentacle).
```

Comment:
Copy-paste mistake? This line seems duplicated.

Reply:
No idea how that happened